### ISSUE TITLE
The minor version should be incremented since execution environments have been changed

### ISSUE THREAD

**Issue by ajohnson1** [eclipse-pde/eclipse.pde#388]
Eclipse Memory Analyzer has a help plugin org.eclipse.mat.ui.help without Java code.
Something changed in Eclipse PDE and now with the MAT projects imported into
a workspace with an API baseline it gets an error:

The minor version should be incremented in version 1.13.0, since execution environments have been changed since version 1.13.0

See [Bugzilla bug 580402](https://bugs.eclipse.org/bugs/show_bug.cgi?id=580402).
As this plugin has no Java code it doesn't have a Bundle-RequiredExecutionEnvironment, so I don't think there should be this error.

The error occurs with eclipse-committers-2021-12 and later but not with eclipse-committers-2021-09 and earlier.

Another project org.eclipse.mat.ui.rcp has a line in MANIFEST.MF 
```Bundle-RequiredExecutionEnvironment: J2SE-1.5```
which has the warning:

Execution environment is lower than one of the plug-in's dependencies (org.eclipse.mat.ui.help) which has an execution 
 environment of JavaSE-17.

Perhaps that is related. Is an omitted Bundle-RequiredExecutionEnvironment allowed?

Copied from [Bugzilla 580980](https://bugs.eclipse.org/bugs/show_bug.cgi?id=580980) as [PDE support](https://www.eclipse.org/pde/support/) said:
>Reporting Bugs
>Submit bugs using Bugzilla web interface.  				
>Please search for duplicates  first! For defects provide steps  				        on how to reproduce. For enhancements outline a use case.   Due to an overwhelming number of requests, committers prioritize  				        critical bug fixes and framework and API improvements and  cannot respond all inquiries.
Submit bugs using [Bugzilla web interface.](https://bugs.eclipse.org/bugs/enter_bug.cgi?product=PDE)

**Comment by merks** [comment]
Does it still happen if you remove the java build and java nature from the .project:

https://git.eclipse.org/c/mat/org.eclipse.mat.git/tree/plugins/org.eclipse.mat.ui.help/.project

These files would then be pointless as well:

https://git.eclipse.org/c/mat/org.eclipse.mat.git/tree/plugins/org.eclipse.mat.ui.help/.settings/org.eclipse.jdt.core.prefs
https://git.eclipse.org/c/mat/org.eclipse.mat.git/tree/plugins/org.eclipse.mat.ui.help/.classpath

Probably this has some pointless entries (no source an bin and no . for a classpath) as well:

https://git.eclipse.org/c/mat/org.eclipse.mat.git/tree/plugins/org.eclipse.mat.ui.help/build.properties

Hopefully that avoids the problem and cleans things up a bit...

**Comment by ajohnson1** [comment]
I have now made those suggested changes, but still get the problem.

**Comment by vik-chand** [comment]
>>it doesn't have a Bundle-RequiredExecutionEnvironment

Did  the baseline have this plugin with BREE ( which is removed now? )

**Comment by vik-chand** [comment]
>>Execution environment is lower than one of the plug-in's dependencies (org.eclipse.mat.ui.help) which has an execution
>>environment of JavaSE-17.

The setting is at Preference->Plug-in Development -> Compilers->Versioning-> Bundle BREE lower than its dependencies

You can set it to Info or Ignore if you dont want it as a warning. This is unrelated issue.

**Comment by vik-chand** [comment]
>clipse Memory Analyzer has a help plugin org.eclipse.mat.ui.help without Java code.
>Something changed in Eclipse PDE and now with the MAT projects imported into
>a workspace with an API baseline it gets an error:

Is there an easy way to recreate this?

**Comment by ajohnson1** [comment]
The baseline plugin did not have a BREE.

I couldn't reproduce the problem on a new workspace by adding an existing Git repo and importing the org.eclipse.mat.help project and then adding my previous installation as a baseline.

I then tried deleting the existing org.eclipse.mat.help project in my old workspace and then reimporting it and the error went away. The project now just has the 'Plug-in development' nature. Perhaps this problem is now solved.

The project doesn't have any Java code, but does provide extensions. Does the API tooling detect extension point (or extension) changes which should cause a change in version number? See https://wiki.eclipse.org/Version_Numbering#When_to_change_the_minor_segment Could the 'API Analysis Nature' do this without the Java nature?
